Bottom: 03d69815a721a0e98bd49a97e7dfe10872cb1b40
Top:    6971bf5ff325b8aa7f48cfba01e20a09d6bd0c82
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-10 17:40:16 +0100

Allow running of action scripts when running PCB as an exporter


---

diff --git a/src/main.c b/src/main.c
index 7e29d56..8b80afe 100644
--- a/src/main.c
+++ b/src/main.c
@@ -998,12 +998,6 @@ main (int argc, char *argv[])
       LayerStringToLayerStack (Settings.InitialLayerStack);
     }
 
-  if (gui->printer || gui->exporter)
-    {
-      gui->do_export (0);
-      exit (0);
-    }
-
   /*    FIX_ME
      LoadBackgroundImage (Settings.BackgroundImage); */
 
@@ -1036,6 +1030,12 @@ main (int argc, char *argv[])
       hid_parse_actions (Settings.ActionString);
     }
 
+  if (gui->printer || gui->exporter)
+    {
+      gui->do_export (0);
+      exit (0);
+    }
+
 #if HAVE_DBUS
   pcb_dbus_setup();
 #endif
